mantidproject/mantid
# JSON Geometry Parser for Nexus-style Geometry

**Description of work.**

This PR concerns the implementation of a geometry parser for the Nexus kafka stream. The geometry will be in JSON format having the nexus file structure  (and classes) superimposed. 

**To test:**

- Through code review
- Ensure unit tests are sensible and pass.
- Ensure performance test is passes and performance is acceptable.

Fixes #26213 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
